docs/quality-assurance/test-tooling.md

```diff
@@ -30,3 +32,65 @@
 |Jmeter|Load test functional behaviour and measure performance|https://jmeter.apache.org/||
 |REST-assured|Testing and validating REST services|http://rest-assured.io/||
 |swagger-request-validator|A Java library for validating request/responses against a OpenAPI / Swagger specification|https://bitbucket.org/atlassian/swagger-request-validator||
+
+## Browser Automation - what tools to use?
+This section will cover the browser automation tools we use at the UKHO and the recommended uses for each one. 
+
+The 3 recommended tools for Browser Automation (UI, e2e and Integration) at the UKHO are:
+
+### Cypress
+Cypress is a front end testing tool built for testing web apps. Cypress test code is executed directly within the browser as JavaScript, meaning there are no language or driver bindings and the tests can execute much faster and with more reliability. It also has full dom and network traffic recording to aid debugging within the Cypress Test Runner. 
+
+Cypress, however, does have its limitations. It has fundamently been created to test your application and your application only, and works best if the Application-Under-Test is a Single Page Application or a Multi Page Application in the same domain that does not utilise pop-ups or new tabs / windows.
+
+Some authentication scenarios can be worked out, primarly SSO and NTLM Windows Auth, but not MSAL Azure B2C Auth.
+
+### Playwright
+Playwright enables fast, reliable and capable automation across all modern browsers. It is a Node.js library to automate Chromium, Firefox and WebKit with a single API.
+
+One of the big disadvantages over Cypress is that it does not have a Test Runner therefore screenshots, videos, dom and network traffic are not recorded out the box, so debugging tests is not as intuitive.
+
+The core advantage of using Playwright over Cypress is it is an out-of-process automation driver that is not limited by the scope of in-page JavaScript execution and can automate scenarios with multiple pages.
+
+Playwrights documentation and community is also not as comprehensive as Cypress or Selenium, yet, it is constantly being added to as the tool is developed and the community is growing.
+
+### Selenium
+Selenium is a tool for web browser automation that uses WebDrivers to remotely control browser instances and emulate a user’s interaction with the browser. One of the reasons that Selenium is popular is because of its available language bindings, these include Java, C#, JavaScript and Python - whereas Cypress and Playwright are JavaScript based.
+
+Selenium is a very good tool to automate true E2E and user interactions, but it takes time and effort to get the framework right, and the overhead of maintaining this along with the WebDrivers and browser versions. It is also not as fast or intuitive as Cypress or Playwright.
```

Comment:
.." and then the overhead of maintaining it along with the WebDrivers and bowser versions" reads better